projects/step-frontend/src/lib/modules/context-menu/resources-list/resources-list.component.html

```diff
@@ -0,0 +1,59 @@
+<section class="users-header">
+  <step-btn-group>
+    <button
+      type="button"
+      mat-raised-button
+      color="primary"
+      [disabled]="!('user-write' | hasRight | async)"
+      (click)="createResource()"
+    >
+      New Resource
+    </button>
+  </step-btn-group>
+</section>
+<step-table [dataSource]="dataSource" matSort [matSortDisableClear]="true">
+  <ng-container matColumnDef="name" stepSearchCol>
+    <th mat-header-cell *matHeaderCellDef mat-sort-header>Name</th>
+    <td mat-cell *matCellDef="let element">
+      <a href="#" (click)="$event.preventDefault(); editResource(element)">{{ element.attributes.name }}</a>
```

Comment:
As alternative, we can prevent link navigation if the event handler will return `false`.
So the `editResouce` itself can return false, or otherwise if we don't want to change method's return type, we can write it as
```
<a href="#" (click)="editResource(element);false">
{{eleement.attributes.name}}
</a>
```

Reply:
Thank Vlad for the hint! Although both of the suggested alternatives have less code, I find the explicit version with `preventDefault()` still easiest to understand. E.g. a new coder in the project works on this code and wonders why this method or expression returns `false`, he wants to clean up and refactors it. In contrast, `preventDefault()` gives him a clue why this piece of code is here.

Reply:
I would prefere  `[routerLink]=""` instead of `href="#"`

Reply:
That is weird, routerLink should work for Angular 2+, in AngularJS you had to do  `href="#" ng-click="$event.preventDefault();`

Reply:
@neogucky As i remember, we decided not to migrate the routing for now.
[routerLink] make sense only in that case, if you want to make a navigation through the Angualr 2+ route
I don't see anything critical in `<a href="#">` in Angular 2+, when it plays the role of button that does some action. 
The only thing the we need to do in such cases, to prevent the original even behavior